File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Pfes/PFE7001Q01LayDailyReportInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Pfes
{
    [Serializable]
    public class PFE7001Q01LayDailyReportInfo
    {
        private String _gumsaDate;
        private String _ioGubun;
        private String _bunho;
        private String _suname;
        private String _gwa;
        private String _doctorName;
        private String _jundalPart;
        private String _hangmogCode;
        private String _hangmogName;

        public String GumsaDate
        {
            get { return this._gumsaDate; }
            set { this._gumsaDate = value; }
        }

        public String IoGubun
        {
            get { return this._ioGubun; }
            set { this._ioGubun = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String JundalPart
        {
            get { return this._jundalPart; }
            set { this._jundalPart = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public PFE7001Q01LayDailyReportInfo() { }

        public PFE7001Q01LayDailyReportInfo(String gumsaDate, String ioGubun, String bunho, String suname, String gwa, String doctorName, String jundalPart, String hangmogCode, String hangmogName)
        {
            this._gumsaDate = gumsaDate;
            this._ioGubun = ioGubun;
            this._bunho = bunho;
            this._suname = suname;
            this._gwa = gwa;
            this._doctorName = doctorName;
            this._jundalPart = jundalPart;
            this._hangmogCode = hangmogCode;
            this._hangmogName = hangmogName;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/pfes/PFE7001Q01LayDailyReportInfo.java
package nta.med.data.model.ihis.pfes;

public class PFE7001Q01LayDailyReportInfo {
	private String gumsaDate   ;
	private String ioGubun     ;
	private String bunho        ;
	private String suname       ;
	private String gwa          ;
	private String doctorName  ;
	private String jundalPart  ;
	private String hangmogCode ;
	private String hangmogName ;
	public PFE7001Q01LayDailyReportInfo(String gumsaDate, String ioGubun,
			String bunho, String suname, String gwa, String doctorName,
			String jundalPart, String hangmogCode, String hangmogName) {
		super();
		this.gumsaDate = gumsaDate;
		this.ioGubun = ioGubun;
		this.bunho = bunho;
		this.suname = suname;
		this.gwa = gwa;
		this.doctorName = doctorName;
		this.jundalPart = jundalPart;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
	}
	public String getGumsaDate() {
		return gumsaDate;
	}
	public void setGumsaDate(String gumsaDate) {
		this.gumsaDate = gumsaDate;
	}
	public String getIoGubun() {
		return ioGubun;
	}
	public void setIoGubun(String ioGubun) {
		this.ioGubun = ioGubun;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getJundalPart() {
		return jundalPart;
	}
	public void setJundalPart(String jundalPart) {
		this.jundalPart = jundalPart;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	
}
